lib/oeqa/selftest/updater.py

```diff
@@ -339,11 +296,9 @@ class HsmTests(oeSelfTest):
 
     @classmethod
     def setUpClass(cls):
-        bb_vars = get_bb_vars(['SYSROOT_DESTDIR', 'base_prefix', 'libdir', 'bindir'],
-                              'aktualizr-native')
-        cls.sysroot = bb_vars['SYSROOT_DESTDIR'] + bb_vars['base_prefix']
-        cls.sysrootbin = bb_vars['SYSROOT_DESTDIR'] + bb_vars['bindir']
-        cls.libdir = bb_vars['libdir']
+        logger = logging.getLogger("selftest")
+        logger.info('Running bitbake to build aktualizr-native tools')
+        bitbake('aktualizr-native')
```

Comment:
Is this extra bitbake necessary here? Running `qemu_launch` calls `bitbake core-image-minimal`, and that should already include `aktualizr-native`.

Reply:
Oh I see, that's why you removed this line in your commit. Thanks!